posthog/urls.py

```diff
@@ -92,6 +93,32 @@ def setup_admin(request):
         })
         return redirect('/')
 
+def social_create_user(strategy, details, backend, user=None, *args, **kwargs):
+    if user:
+        return {'is_new': False}
+
+    signup_token = strategy.session_get('signup_token')
+    if signup_token is None:
+        processed = render_to_string('auth_error.html', {'message': "You're attempting to create an account without a team. Please use an invite link from the team you want to join!"})
+        return HttpResponse(processed, status=401)
+
+    fields = dict((name, kwargs.get(name, details.get(name)))
+                   for name in backend.setting('USER_FIELDS', ['email']))
+    
+    if not fields:
+        return
+
+    user = strategy.create_user(**fields)
+    team = Team.objects.get(signup_token=signup_token)
```

Comment:
Would be good to catch `Team.DoesNotExist` and error into "We can't find the team associated with this signup token"